src/test/java/com/epam/tc/hw7/enteties/MetalColor.java

```diff
@@ -0,0 +1,82 @@
+package com.epam.tc.hw7.enteties;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MetalColor {
```

Comment:
As improvement you can extend `com.jdiai.tools.DataClass` like 
```suggestion
public class MetalColor extends DataClass<MetalColor> {
```
benefits of this approach can be found in [documentaton](https://jdi-docs.github.io/jdi-light/#standard-login-form)

\+ same for `User` class

Reply:
Extended, read about benefits